packages/core/src/test/codewhisperer/testUtil.ts

```diff
@@ -32,7 +32,8 @@ export async function resetCodeWhispererGlobalVariables() {
     CodeWhispererCodeCoverageTracker.instances.clear()
     globals.telemetry.logger.clear()
     session.reset()
-    await CodeSuggestionsState.instance.setSuggestionsEnabled(false)
+    await globals.globalState.clear()
+    await CodeSuggestionsState.instance.setSuggestionsEnabled(true)
```

Comment:
`true` is the default since https://github.com/aws/aws-toolkit-vscode/commit/2fed6a62288c291d22b6555abd4b23569fc31c3e . This wasn't caught because the tests use stubs/mocks. This is an example of why stubs/mocks are fragile and don't represent the actual application code.